themes/default/content/what-is/what-is-secrets-management.md

```diff
@@ -0,0 +1,57 @@
+---
+title: What Is Secrets Management?
+meta_desc: |
+    Understand secrets management, the importance of secrets management, and how secrets management relates to infrastructure as code and configuration management
+type: what-is
+page_title: What is secrets management?
```

Comment:
Super minor thing, but we've been writing out the `page_title` sans "?". For consistency, consider removing it (I wonder though if perhaps we should be including it in all the pages.)

Reply:
Let me check all the other pages and see what the most common approach is. Unless folks feel strongly one way or another, we can adopt whatever approach the majority of the pages is using. Fair?

Reply:
Of the 7 pages that ask "What is...", 5 of them include a question mark in the `page_title`.

That being said, I'm happy to go whatever direction folks would prefer. :-)

Reply:
Looks like team consensus via Slack is to keep the question mark.